avoid error:
error: 'environ' undeclared (first use in this function)

diff --git a/src/mate-screensaver-preferences.c b/src/mate-screensaver-preferences.c
index 3c7621a..56685a4 100644
--- a/src/mate-screensaver-preferences.c
+++ b/src/mate-screensaver-preferences.c
@@ -25,7 +25,6 @@
 
 #include "config.h"
 
-#define _GNU_SOURCE
 #include <unistd.h>
 #include <stdlib.h>
 #include <string.h>
